rero_ils/modules/items/api_views.py

```diff
@@ -282,7 +282,7 @@ def item(item_barcode):
         abort(404)
     loan = get_loan_for_item(item.pid)
     if loan:
-        loan = Loan.get_record_by_pid(loan['loan_pid']).dumps_for_circulation()
+        loan = Loan.get_record_by_pid(loan.get('pid')).dumps_for_circulation()
```

Comment:
loan.pid ?

Reply:
it is an invenio_circulation loan class which does not have the pid shortcut